test/precompile.jl

```diff
@@ -1044,6 +1065,11 @@ precompile_test_harness("code caching") do dir
         m = only(methods(MC.call_buildstale))
         mi = m.specializations::Core.MethodInstance
         @test hasvalid(mi, world)       # was compiled with the new method
+        m = only(methods(MA.fib))
+        mi = m.specializations::Core.MethodInstance
+        @test isdefined(mi, :cache)     # it was precompiled by StaleB
+        @test !hasvalid(mi, world)      # invalidated by redefining `gib` before loading StaleB
+        @test MA.fib() === 2.0
```

Comment:
```suggestion
        @test_broken !hasvalid(mi, world)      # invalidated by redefining `gib` before loading StaleB
        @test_broken MA.fib() === 2.0
```

Reply:
Meaning, you want to see this merged and then fixed in a separate PR?